schema/examples/direct/multiple_signers/status-response.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<direct-signature-job-status-response xmlns="http://signering.posten.no/schema/v1">
+    <signature-job-id>1</signature-job-id>
+    <status signer="12345678910">SIGNED</status>
+    <status signer="10987654321">SIGNED</status>
+    <confirmation-url>https://api.signering.posten.no/api/{sender-identifier}/direct/signature-jobs/1/complete</confirmation-url>
+    <xades-url signer="12345678910">https://api.signering.posten.no/api/{sender-identifier}/direct/signature-jobs/1/xades/1</xades-url>
+    <xades-url signer="10987654321">https://api.signering.posten.no/api/{sender-identifier}/direct/signature-jobs/1/xades/2</xades-url>
+    <pades-url>https://api.signering.posten.no/api/{sender-identifier}/direct/signature-jobs/1/pades</pades-url>
+</direct-signature-job-status-response>
```

Comment:
`status` vil i tillegg kunne ha `EXPIRED` (introdusert ifm. polling) ved én signatar.
De nye statusene _trenger_ ikke å opptre for enkeltsignatarer (selv om `FAILED` kan gjøre det i det nåværende forslaget), så i teorien kan det funke.

Men selv om det "kan funke" føles det litt rart å enten sette inn en signatar-status _eller_ oppdrags-status i `status`-feltet avhengig av om det er et enkel- eller multisignatar-oppdrag.

Jeg verifiserer responsvalidering.